drivers/media/xen/xen_camera_front_v4l2.c

```diff
@@ -805,6 +819,13 @@ int ioctl_enum_frameintervals(struct file *file, void *fh,
 	struct xen_camera_front_cfg_format *format;
 	struct xen_camera_front_cfg_resolution *resolution;
 
+	/*
+	 * Emulate what __video_do_ioctl() in v4l2-ioctl.c returns for invalid
```

Comment:
Do not put any statement that we emulate something. Just state that if no frame rates are supported it is equivalent to the corresponding ioctl is not supported and this can be handled by user-space. Otherwise, during frame rates enumeration, returning -EINVAL on the very first ioctl may not be handled by user-space properly.

Reply:
will update